include/zephyr/xen/public/grant_table.h

```diff
@@ -0,0 +1,397 @@
+/******************************************************************************
```

Comment:
One of the license issues may be there is no `SPDX-License-Identifier:` I think

Reply:
Yes I just saw that other public headers in zephyr/xen/public starts with this:

> /* SPDX-License-Identifier: MIT */
> 
> [original header from xen]

Reply:
Fixed.